base/ltclass.dtx

```diff
@@ -33,7 +33,7 @@
 %<*driver>
 % \fi
 \ProvidesFile{ltclass.dtx}
-             [2021/03/05 v1.3z LaTeX Kernel (Class & Package Interface)]
+             [2021/03/27 v1.3aa LaTeX Kernel (Class & Package Interface)]
```

Comment:
What version should this be?  It's not really a big change to deserve an increase to 1.4, but we've run short of letters in the alphabet :-)

Reply:
Historically we just moved to 1.4a (regardless of the size of change) so I would continue doing that here too.